src/main/EmailsDataBase.java

```diff
@@ -0,0 +1,81 @@
+package main;
+
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.*;
```

Comment:
It's better not to use wildcards, please configure your IDEA settings accordingly